index.d.ts

```diff
@@ -15,6 +15,11 @@ export interface ProgressData {
 	Completed percentage. A value between `0` and `1`.
 	*/
 	readonly percent: number;
+
+	/**
+	Path of deleted file
```

Comment:
From onProgress docs:

> Called after each file or directory is deleted.

So it could be a directory too.

---

It would also be useful to make it clear it's an "absolute path".